### ISSUE TITLE
news_categories not working with dc_multilingual

### ISSUE THREAD

**Issue by asaage** [codefog/contao-news_categories#139]
used versions:
dc_multilingual 3.0.2
contao-news_categories Version 3.0.8
contao 4.5.6

```log
An exception occurred while executing 'INSERT INTO tl_news_category (`pid`, `sorting`, `tstamp`) VALUES (0, 128, 0)': SQLSTATE[HY000]: General error: 1364 Field 'language' doesn't have a default value
```

**Comment by qzminski** [comment]
Fixed in b496750. The new hotfix version should be released in a couple of minutes.

**Comment by asaage** [comment]
After applying this update i get:
```
An exception occurred while executing 'INSERT INTO tl_news_category (lid,tstamp,language,pid) VALUES ('3',1521580642,'en','1')': SQLSTATE[HY000]: General error: 1364 Field 'sorting' doesn't have a default value
```

**Comment by qzminski** [comment]
True, fixed in 0ce6ed5. Can you try installing the `hotfix/3.0.10` branch before I release a new version?
